spec/models/alarm_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+RSpec.describe Alarm, type: :model do
+  subject { FactoryBot.create(:alarm) }
+
+  describe 'all attributes' do
+    it 'should be valid' do
+      expect(subject).to be_valid
+    end
+  end
+
+  describe 'validations' do
+    it 'validate presence of required fields' do
+      should validate_presence_of(:days)
+      should validate_presence_of(:time)
+      should validate_presence_of(:user_id)
```

Comment:
You can do to be consistent with a syntax. 
```
expect(subject).to validate_presence_of(:your_attribute) 
```

Reply:
Same here:
```
 it { expect(subject).to belong_to(:your_model) }
```

Reply:
Thanks for the comment, It is solved